resources/views/hr/application/index.blade.php

```diff
@@ -46,102 +46,145 @@
         $search = request()->has('search') ? '&search=' . request('search') : '';
         $query_filters = $hr_job_id . $search
     @endphp
-    <div class="d-flex align-items-center justify-content-between">
-        <ul class="nav nav-pills mb-2">
-            <li class="nav-item">
-                <a class="nav-item nav-link d-flex align-items-center {{ $status ? 'text-info' : 'active bg-info text-white' }}"
-                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}}> <i
-                    class="fa fa-clipboard"></i>&nbsp;
+    <div class="d-flex bg-white status-icons ">
+        <ul class="nav mb-2 d-flex justify-content-between">
+            <li class="nav-item mx-3">
+                <a class="nav-item nav-link d-flex flex-column align-items-center px-0 {{!$status || $status === config('constants.hr.status.new.label') ? 'active text-theme-gray-dark' : 'text-theme-gray-light' }}"
+                    href=/{{ Request::path() }}?status={{ config('constants.hr.status.new.label') }}{{$query_filters}}>
+                    <div class="position-relative">
+
+                        <span
+                            class=" d-inline-block py-0"
+                            style="font-family: 'Mulish', sans-serif; font-size: 10px;font-weight:700;line-height:12px;position: absolute;top: -10px;right: -1px;">
+                            {{$newApplicationsCount + $inProgressApplicationsCount - $trialProgramCount}}
```

Comment:
Fromat accordingly at all the other places. Add spaces wherever `{{ }}` are used
```suggestion
                            {{ $newApplicationsCount + $inProgressApplicationsCount - $trialProgramCount }}
```